CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The changes documented here do not include those from the original repository.
 
 ## [Unreleased]
 
+## 2024-03-11
+- Fixed library build (https://outsystemsrd.atlassian.net/browse/RMET-3190).
```

Comment:
Do we need this entry?

Reply:
I was actually wondering the same thing, I'll take it out.